guides/common/modules/proc_applying-remote-scap-resources-in-a-disconnected-environment.adoc

```diff
@@ -2,10 +2,10 @@
 = Applying remote SCAP resources in a disconnected environment
 
 SCAP data streams can contain remote resources, such as OVAL files, that the SCAP client can fetch over the internet when it runs on hosts.
-If your host does not have internet access, you must download remote SCAP resources and distribute them to your host as local files.
+If your hosts do not have internet access, you must download remote SCAP resources and distribute them from {ProjectServer} to your hosts as local files.
```

Comment:
I assume you can also distribute it via Smart Proxy Servers; just as any other synced content.

Reply:
I'm not sure what you mean. You still have to add the content on ProjectServer first?

Reply:
Yes, you'd have to add content to {Project} first. Then, irrespective of whether hosts can access the internet, you distribute OpenSCAP content to your hosts similar to other content.

Reply:
Or does it? :D Ohhh, I think I'm getting an idea what you mean...

Reply:
If I wanted to distribute it via SmartProxy, it would have to be added to a content view etc., right?

Reply:
The latter is what is documented.

Reply:
Actually, neither. When the host cannot access the internet, the file has to be distributed to the host BEFORE the SCAP client runs, because the SCAP client reads local files.

Reply:
OK. Then I'm thinking: In the context of Foreman and centralized infrastructure management: wouldn't you _always_ want to distribute those files via Foreman Server/Smart Proxy Servers as you distribute other content too?

Reply:
Yeah, probably. Fair point.